articles/service-fabric/service-fabric-backup-explorer.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Read and Update Reliable Collections Backup Locally 
+description: This article describes how to read and update Reliable collectios backup locally
+services: service-fabric
+documentationcenter: .net
+author: athinanthny
+manager: chackdan
+
+ms.service: service-fabric
+ms.devlang: dotnet
+ms.topic: article
+ms.date: 2/01/2019
+ms.author: atsenthi
```

Comment:
These are the only required metadata fields (rest are obsolete/redundant to global defaults):

- title
- description
- ms.topic: **conceptual**
- ms.date: **07/02/2020** (or some other recent date)